zapisy/apps/defects/templates/editDefect.html

```diff
@@ -59,30 +59,27 @@ <h1 class="d-inline-block">
     <form action="" method="POST" class="post-form" enctype="multipart/form-data" id="main-form">
         {% csrf_token %}
         {% crispy form %}
-        <input type="submit" hidden />
+
         <hr>
 
         <div id="image-formset" class="mb-5">
             <h4>Zdjęcia</h4>
 
-            {% if edit %}
-                {% if images %}
-                    <div class="container-fluid" style="margin: 10px">
-                        Aby usunąć wybrane zdjęcia zaznacz je za pomocą kwadratu w lewym górnym rogu zdjęcia.
-                        <div class="row">
-                            {% for image in images %}
-                                <div class="col-md-3 center-block defect-edit-image" >
-                                    <label class="delete-label">
-                                        <input type="checkbox" name="files-to-delete[]" value="{{ image.0 }}">
-                                        <span class="checkmark"></span>
-                                    </label>
-                                    <img class="img-thumbnail" src={{ image.1 }} >
-                                </div>
-                            {% endfor %}
-                        </div>
+            {% if edit and images%}
+                <div class="container-fluid" style="margin: 10px">
+                    Aby usunąć wybrane zdjęcia zaznacz je za pomocą kwadratu w lewym górnym rogu zdjęcia.
```

Comment:
```suggestion
                    Aby usunąć wybrane zdjęcia, zaznacz je za pomocą przycisku w lewym górnym rogu miniaturki.
```
… albo jakoś inaczej, mam poczucie, że "zaznacz" to trochę niewłaściwa nazwa czynności, którą tu się wykonuje – ale możliwe, że przesadzam.